data_model.md

```diff
@@ -0,0 +1,449 @@
+
+-   Interval label - Indicates if a time labels the beginning or the
+    ending of an interval average, or indicates an instantaneous value,
+    e.g. beginning, ending, instant
+
+-   Value type - The type of the data in the forecast, e.g. mean, max,
+    95th percentile.
+
+-   Variable - The variable in the forecast, e.g. power, GHI, DNI. Each
+    variable is associated with a standard unit.
+
+-   Site - The predefined site that the forecast is for, e.g. Power
+    Plant X or Aggregate Y.
+
+Additional parameters, such as model configuration parameters, may be
+specified under the “extra parameters” key.
+
+
+Each Forecast has data associated with it. The format of this data is
+found in the Data Format section below
+
+### Aggregates
+{: .anchor}
+
+Each Aggregate must be associated with one or more Observations. Solar
+Forecast Arbiter will calculate aggregated observation data generated
+from these Observations. Each aggregate will also provide a list of
+Sites associated with the Observations. These Sites will provide
+forecast providers with the modeling parameters for each system in the
+Aggregate. Aggregate forecasts will be associated directly with an
+Aggregate. Forecast providers will be required to submit a forecast of
+the aggregated quantity.
+
+Aggregates have the following required parameters:
+
+-   Name: Name of the Aggregate
+
+-   Description: A description of the makeup of the Aggregate, e.g. all
+    utility scale solar in ISO region
+
+-   Timezone: IANA timezone of the site, e.g. Etc/GMT+8
+
+-   Observations: A list of Observations that make up the aggregate
+
+## Variables and Units
+{: .anchor}
+
+Solar Forecast Arbiter will only accept a standard set of variables and
+units. Data not in in the correct units must be converted before it is
+uploaded. The forecast/observation variables and units that are
+permitted are as follows:
+
+-   surface temperature: degrees Celsius
+
+-   surface wind speed: meters per second
+
+-   Global Horizontal Irradiance: Watts per meter squared
+
+-   Direct Normal Irradiance: Watts per meter squared
+
+-   Diffuse Horizontal Irradiance: Watts per meter squared
+
+-   Plane of array irradiance: Watts per meter squared
+
+-   Relative humidity: %
+
+-   AC power: megawatts
+
+-   DC power: megawatts
+
+-   PDF probability: %
+
+-   CDF value:
+
+Timestamp and timezone formatting is addressed below.
+
+Data Formats
+------------
+{: .anchor}
+
+In this section Data is defined as both observation and forecast data.
+The data contain a time series of data points. Each data point has
+fields determined by the data’s type listed below:
+
+#### Observations
+{: .anchor}
+
+-   Timestamp: A timestamp in ISO-8601 format including a timezone, e.g.
+    2018-12-05T23:01:45-07:00 or 20181206T060145Z.
+
+-   Value: Values for the variable in units defined above.
+
+-   Quality Flag: A flag indicating if the value is questionable. Either
+    0 or 1.
```

Comment:
I suggest replacing "Either..." with "Uploads may contain values 0 (ok) or 1 (questionable). Downloads may contain additional flags determined by the data qualification toolkit (discussed elsewhere)." 

cc @alorenzo175 @cwhanse